factory/tests/factory_tests.rs

```diff
@@ -0,0 +1,112 @@
+use factory_io::*;
+use gstd::{prelude::*, ActorId};
+use gtest::System;
+mod utils;
+use utils::*;
```

Comment:
No idiomatic way to use external functions.
https://doc.rust-lang.org/stable/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html
```suggestion
```